src/core/include/units/quantity.h

```diff
@@ -392,25 +404,39 @@ template<Quantity Q1, QuantityEquivalentTo<Q1> Q2>
   return ret(ret(lhs).count() - ret(rhs).count());
 }
 
-template<typename D1, typename U1, typename Rep1, typename D2, typename U2, typename Rep2>
-  requires quantity_value_for_<std::multiplies<>, Rep1, Rep2>
-[[nodiscard]] constexpr Quantity auto operator*(const quantity<D1, U1, Rep1>& lhs, const quantity<D2, U2, Rep2>& rhs)
+template<typename QuantityOrQuantityValue, typename D, typename U>
+  requires Quantity<QuantityOrQuantityValue> || QuantityValue<QuantityOrQuantityValue>
```

Comment:
Can we get rid of `quantity` support on `lhs`?

Reply:
Of course. I'll do it as I said above. I just wanted to clear up all other conversations so I could focus on what's left, so I submitted what I've done so far.

Reply:
Of course. I'll do it as I said above. I just wanted to clear up all other conversations so I could focus on what's left, so I submitted what I've done so far.